lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -352,10 +353,14 @@ PtrTy RawMemManager::ptrAdd(PtrTy ptr, Expr offset) const {
 }
 
 /// \brief Stores an integer into memory
-///
+/// \param[in] ptr is the address at which _val will be stored
+/// \param[in] _val is the value being written
+/// \param[in] mem is the memory bank/register being written to
+/// \param[in] byteSz is the size of _val in bytes (should be =< word size)
+/// \param[in] align is the known alignment of the ptr
 /// Returns an expression describing the state of memory in \c memReadReg
 /// after the store
-/// \sa loadIntFromMem
```

Comment:
typo. should be loadIntFromMeme

Reply:
The original comment is wrong. It says loadIntFromMem for storeIntToMem